net.bobbo.player-controller/player_scripts/crouch.gd

```diff
@@ -14,11 +14,18 @@ var crouch_current: float = 0
 var crouch_target: float = 0
 const crouch_action: String = "player_crouch"
 
+## max_player_height is needed for proper uncrouching height calculations if the maximum player height were to change during gameplay at any point this value would need to somehow be updated to account for that default is 2 because the default height in player_controller is 2
```

Comment:
So wait - is this the maximum possible player height, or is this the height of the player in normal conditions?

Reply:
Looking at & thinking about this caused me to open ticket https://github.com/Iseeicy/netengine5/issues/44 . Let's tackle that after this PR is squeaky clean!